api/users/serializers.py

```diff
@@ -444,6 +449,20 @@ def verify_two_factor(self, instance, value, two_factor_addon):
             raise exceptions.PermissionDenied(detail='The two-factor verification code you provided is invalid.')
         two_factor_addon.save()
 
+    def request_deactivation(self, instance, requested_deactivation):
+        if instance.requested_deactivation != requested_deactivation:
+            if requested_deactivation:
+                mails.send_mail(
+                    to_addr=OSF_SUPPORT_EMAIL,
+                    mail=mails.REQUEST_DEACTIVATION,
+                    user=instance,
+                    can_change_preferences=False,
+                )
+                instance.email_last_sent = timezone.now()
```

Comment:
We're not using this field to throttle but it still doesn't hurt to update this value.